### DISCUSSION TITLE
Haven't received the result of the Campus Experts Application.

### DISCUSSION THREAD

**Discussion by nityaa3009** [community/community#145155] [GitHub Education]
Hey!
So, I was supposed to get the final result of the Campus Experts application on the 21st November but I still haven't received any mail from them. What do I do? Does it mean that my application has been rejected?

**Comment by davevad93** [comment]
Hi @nityaa3009 ,

If you live in another timezone where it's already the 21st, try to wait a few hours because on the U.S. East Coast it's still 8:00 PM on November 20.

**Reply by nityaa3009** [reply]
Yeah, I am in a different timezone right now. The thing is that the last time the mail had to come, it came a day early and so I thought that'd be the case this time too.
Hopefully I get it on time!
